README.md

```diff
@@ -71,22 +71,22 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+this definitely seems like a model method to me. separation of concerns.
 ```
 
 ### Question 7
 
 You clone yet another Tunr repo. Put the following commands in the correct order necessary to make the app run. Delete the one command that will not be used.
 
 ```
-$ rake db:drop
-$ rake db:create
+$ git clone git@github.com:ga-wdi-exercises/moar-tunr.git
 $ rails new . -d postgresql
 $ bundle install
-$ git clone git@github.com:ga-wdi-exercises/moar-tunr.git
+$ rake db:create
 $ rake db:migrate
-$ rails s
 $ rake db:seed
+$ rails s
+$$$$$$$$$$$$
 ```
 
 ### Question 8
```

Comment:
It doesn't validate because of the ERB tags. If you wanted to test validating it, though, you could copy the source code when viewing your app from the browser and validate that instead.